config/testgrid/dashboards.yaml

```diff
@@ -4,16 +4,12 @@ dashboard_groups:
   dashboard_names:
   - jetstack-cert-manager-master
   - jetstack-cert-manager-previous
-  - jetstack-cert-manager-next
   - jetstack-cert-manager-presubmits-blocking
-  - jetstack-cert-manager-website
```

Comment:
We no longer should have any ProwJobs for cert-manager website that would be marked to be displayed by TestGrid since we removed the algolia check https://github.com/jetstack/testing/pull/633